frontend/ts/announcements.ts

```diff
@@ -70,6 +70,7 @@ document.announcements = (() => {
     };
 
     setInterval(fetchAnnouncements, 10 * 1000);
+
```

Comment:
😁 pls revert too

Reply:
@natsukagami done in [40d3a00](https://github.com/natsukagami/kjudge/pull/96/commits/40d3a003354ee632d49f90cbfba1583a98359f2a) :)